library/websphere-liberty

```diff
@@ -11,60 +11,143 @@ Directory: beta
 Tags: kernel
 Directory: ga/latest/kernel
 
+Tags: kernel-java11
+Directory: ga/latest/kernel
+Architectures: amd64, ppc64le, s390x
```

Comment:
Unless you add a `File: Dockerfile.java11`, this will just be tagging the `kernel` image as `kernel-java11` for the 3 architectures (same goes for all the new java11 images below).

Reply:
thanks for pointing that out @yosifkit. I've updated my PR accordingly